build/ci/e2e/Jenkinsfile-new

```diff
@@ -0,0 +1,79 @@
+pipeline {
+
+    agent {
+        label 'linux'
+    }
+
+    options {
+        timeout(time: 150, unit: 'MINUTES')
+    }
+
+    environment {
+        VAULT_ADDR = credentials('vault-addr')
+        VAULT_ROLE_ID = credentials('vault-role-id')
+        VAULT_SECRET_ID = credentials('vault-secret-id')
+        GCLOUD_PROJECT = credentials('k8s-operators-gcloud-project')
+        CLUSTER_VERSION = '1.12'
+        CLUSTER_NAME = "${BUILD_TAG}"
+    }
+
+    stages {
+        stage('Checkout from GitHub') {
+            steps {
+                checkout scm
+            }
+        }
+        stage("Run E2E tests") {
+            steps {
+                sh """
+                    cat >operators/run-config.yml <<EOF
+id: gke-ci
+overrides:
+  kubernetesVersion: $CLUSTER_VERSION
+  clusterName: $CLUSTER_NAME
+  vaultAddress: $VAULT_ADDR
+  vaultRoleId: $VAULT_ROLE_ID
+  vaultSecretId: $VAULT_SECRET_ID
+  gke:
+    gcloudProject: $GCLOUD_PROJECT
+EOF
+                    sh 'make -C build/ci ci-e2e-new'
```

Comment:
Either:
```
sh """
bla bla bla
"""
sh 'make -C build/ci ci-e2e-new'
```
or 
```
sh """
bla bla bla
make -C build/ci ci-e2e-new
"""
```
Otherwise it will fail